public/data/abdussalam_02.json

```diff
@@ -41,14 +41,14 @@
     {
       "title": "Education",
       "date": "June 2021",
-      "icon": "Graduation Hat",
+      "icon": "graduation-hat",
       "color": "grey",
       "description": "Pulling of my Bachelors in Computer Application from IGNOU"
     },
     {
       "title": "Tech Journey",
       "date": "January 2018",
-      "icon": "DEV.to",
+      "icon": "dev.to",
```

Comment:
I thought the case didn't matter?

Reply:
doesn't matter either, I thought this might fix it because it has a `.` but didn't 😅